packages/sonarwhal/.sonarwhalrc

```diff
@@ -35,6 +35,7 @@
         "no-protocol-relative-urls": "warning",
         "no-vulnerable-javascript-libraries": "warning",
         "performance-budget": "warning",
+        "stylesheet-limits": "warning",
```

Comment:
?

Reply:
I copied the `.sonarwhalrc` from master branch and pasted in my version.  anyway, removed this new rule now

Reply:
@alrra Thanks! Pushed another commit which fixes all these small issues.